robottelo/performance/pulp.py

```diff
@@ -72,19 +74,19 @@ def get_enabled_repos(org_id):
         """
         LOGGER.info('Searching for enabled repositories by hammer CLI:')
 
-        result = Repository.list(
-            {'organization-id': org_id},
-            per_page=False
-        )
-
-        if result.return_code != 0:
+        try:
+            result = Repository.list(
+                {'organization-id': org_id},
+                per_page=False,
```

Comment:
ah! robottelo handles this attribute as boolean - ok.  since hammer cli handles this as an integer this is confusing.  Thank you for the clarification